deployments/helm/dra-example-driver/templates/kubeletplugin.yaml

```diff
@@ -58,7 +58,7 @@ spec:
               fieldPath: metadata.namespace
         # Simulated number of devices the example driver will pretend to have.
         - name: NUM_DEVICES
-          value: "8"
+          value: "9"
```

Comment:
If changing the default here means we can skip changing the default for the command line flag in the binary then I'd say let's keep the default of 8 in the binary just to avoid any churn there that we can. We could do that as a follow-up.